apps/web/app/api/(internal)/insights/lib/utils.ts

```diff
@@ -96,7 +97,7 @@ export const generateInsightsEnabledForSurveyQuestions = async (
       return question;
     });
 
-    const updatedSurvey = await prisma.survey.update({
+    await prisma.survey.update({
```

Comment:
Any specific reasons for not using our general services here, as in the end we are returning `TSurvey`, right now we do two prisma get calls (one using general service and one with custom fetching) and one prisma update call. 

Instead can we follow this flow

Get survey using getSurvey
Update it using updateSurvey (const updatedSurvey = await updateSurvey(newSurvey))
return updatedSurvey